### DISCUSSION TITLE
Multi scores don't update user stats

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#15887] [Q&A] [answered]
When in a multi room, after playing a map the stats on your profile, like playcount, don't update, like they do on solo play.

logs:
[performance.log](https://github.com/ppy/osu/files/7632733/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7632735/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7632736/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7632737/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7632738/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7632739/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7632740/updater.log)
[database.log](https://github.com/ppy/osu/files/7632741/database.log)
[network.log](https://github.com/ppy/osu/files/7632748/network.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15888
